jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/kafka/producer/LoggingProducerListener.java

```diff
@@ -22,8 +22,8 @@ public final class LoggingProducerListener<K, V>
 
   @Override
   public void onSuccess(ProducerRecord<K, V> producerRecord, RecordMetadata recordMetadata) {
-    log.debug("Successfully produced key {} and value {} to topic {}", producerRecord.key(),
-        producerRecord.value(), producerRecord.topic());
+    log.debug("Successfully produced to topic with {} key {} and value {} ", producerRecord.topic(),
```

Comment:
I feel like you could switch this debug log around to make it read a little better. You actually have it this way in another debug log in the same file.

```suggestion
    log.debug("Successfully produced to topic {} with key {} and value {} ", producerRecord.topic(),
```